software/Test_Robot_Software/mainboard_communication.py

```diff
@@ -0,0 +1,32 @@
+from api import RobotMovement
+from robot_state import State
+
+class MainboardComm:
+    """
+    This class is there just for veryfying everythin works
+    whena speed command is sent to the MCU
+    via serial
+    """
+
+    def __init__(self):
+        self.current_state = State.MANUAL
```

Comment:
The state should be defined in one place, not spread across multiple classes and files